src/core/components/providers/markdown.jsx

```diff
@@ -40,20 +40,8 @@ Markdown.propTypes = {
 
 export default Markdown
 
-const sanitizeOptions = {
```

Comment:
So all these "fancy" options we had are all covered by default on DomPurify?
I guess if all the Unit Test pass we are good, right?

Reply:
@heldersepu yeah, looks like this library tends to have a lighter touch 😄